mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -96,10 +94,12 @@ func ignitionFailure(c cluster.TestCluster) error {
 	}()
 	go func() {
 		time.Sleep(2 * time.Minute)
-		proc := os.Process{
-			Pid: inst.Pid(),
+		if inst != nil {
```

Comment:
Thinking more on this, I think a cleaner approach is to restructure this code so that we just have one goroutine (the one above that does `WaitAll()`), and then we `select` between `errchan` and the 2 minute timeout?

This is OK too though; it just feels funny that in the happy case we're leaving a goroutine running which will eventually timeout.

Reply:
Right.  Actually one thing I am just hitting now is we wait *forever* inside the qemu flight code if ssh doesn't come up, which is related and super broken...